app/views/organizations/staff/manage_fosters/_foster_cards.html.erb

```diff
@@ -0,0 +1,73 @@
+<% foster_pets.each do |pet, fosters| %>
+  <%# Grid cell %>
+  <div class="col-lg-4 col-md-6 p-0">
```

Comment:
I think we have multiple card partials at this point. It may be a good task for the mobile work for the conference to make a shared card partial/component.